git/github/github.go

```diff
@@ -232,6 +234,7 @@ func (c *Client) CreateCheckRun(ctx context.Context, cra *CheckRunAdapter) (*int
 		Name:       cra.Name,
 		HeadSHA:    cra.SHA,
 		ExternalID: &cra.ExternalID,
+		DetailsURL: &cra.DetailsURL,
```

Comment:
I think this line should be removed in light of the added logic on the lines below ([current link](https://github.com/redhat-appstudio/integration-service/pull/651/files#diff-9896ed3f1575fe48878f4f5d1b4d39680607d849a5fa3604fe745f0ab79efbb4R256-R260)), that only sets the `detailsURL` if `cra.DetailsURL` is not empty.